lib/matplotlib/mathtext.py

```diff
@@ -1824,19 +1819,19 @@ def __init__(self, state):
 class Glue(Node):
     """
     Most of the information in this object is stored in the underlying
-    :class:`GlueSpec` class, which is shared between multiple glue objects.  (This
-    is a memory optimization which probably doesn't matter anymore, but it's
-    easier to stick to what TeX does.)
+    :class:`GlueSpec` class, which is shared between multiple glue objects.
+    (This is a memory optimization which probably doesn't matter anymore, but
+    it's easier to stick to what TeX does.)
     """
     def __init__(self, glue_type, copy=False):
         Node.__init__(self)
         self.glue_subtype   = 'normal'
-        if isinstance(glue_type, six.string_types):
+        if isinstance(glue_type, str):
             glue_spec = GlueSpec.factory(glue_type)
         elif isinstance(glue_type, GlueSpec):
             glue_spec = glue_type
         else:
-            raise ValueError("glue_type must be a glue spec name or instance.")
+            raise ValueError("glue_type must be a glue spec name or instance")
```

Comment:
Any particular reason for getting rid of the full stop?

Reply:
Error messages (i.e. arguments to exceptions) typically don't contain a full stop, even if they are otherwise full sentences.  e.g.
```
In [2]: f(1)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-281ab0a37d7d> in <module>()
----> 1 f(1)

TypeError: f() takes 0 positional arguments but 1 was given
```
and Matplotlib mostly follows the same convention (no stats, just an empirical observation).